src/lib/eZ/RichText/Converter/Render/Template.php

```diff
@@ -115,6 +117,21 @@ protected function processTemplate(DOMDocument $document, DOMXPath $xpath, DOMEl
         }
     }
 
+    /**
+     * Provides attributes map in format:
+     * [
+     *     attribute => param,
+     *     ...
+     * ].
```

Comment:
In case of `array` return or argument for scalar (or any non-nested structure) values, it is good practice to provide a docblock type notation like: `@return string[]`.